bin/cheribsdtest/cheribsdtest_cheriabi.c

```diff
@@ -284,6 +284,22 @@ CHERIBSDTEST(cheriabi_munmap_invalid_ptr,
 	cheribsdtest_success();
 }
 
+CHERIBSDTEST(cheriabi_mprotect_upgrade_prot,
+    "Check that upgrading from PROT_NONE works")
```

Comment:
Do you want to mention "cap" or capability in the description?  Maybe "Check that upgrading from PROT_NONE includes capability permissions"?